src/main/java/org/eeichinger/servicevirtualisation/jdbc/XmlTypeRegistry.java

```diff
@@ -0,0 +1,49 @@
+package org.eeichinger.servicevirtualisation.jdbc;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+
+public class XmlTypeRegistry {
+
+    private static final List<XmlTypeInfo<?>> DEFAULT_TYPES = new ArrayList<>();
+    static {
```

Comment:
I just added the ones we were using for now. It's easy to extend when using, but it would also be easy to add more defaults if someone were so inclined.